src/Storefront/Resources/app/storefront/src/plugin/header/account-menu.plugin.js

```diff
@@ -17,9 +17,9 @@ export default class OffCanvasAccountMenu extends Plugin {
         additionalClass: 'account-menu-offcanvas',
 
         /**
-         * class is used to hide the dropdown on viewports where the offcanvas is used
+         * class is used to show the dropdown on viewports where the offcanvas is not used
          */
-        hiddenClass: 'd-none',
```

Comment:
@tobiasberge Do you think this is considered a break?
If not, I'd approve this MR :slightly_smiling_face:

Reply:
Strictly speaking this would be a break, because the plugin option `hiddenClass` is now gone and has no effect anymore if used e.g. to pass a custom CSS class.

We could leave the "old" `hiddenClass` option as well as the classList assignment alongside with the new `showClass`. Then there would be some "dead" code but we avoid the breaking change:

```js
/** @deprecated tag:v6.7.0 - Use "showClass" instead. */
hiddenClass: 'd-none',

showClass: 'show',

/*
 ----
*/

/** @deprecated tag:v6.7.0 - "hiddenClass" is deprecated. "showClass" will be used instead. */
this._dropdown.classList.remove(this.options.hiddenClass);
this._dropdown.classList.add(this.options.showClass);
```
----

Coincidentally I was also working on this ticket. 😆

I had a different approach and tried to omit the opening of the dropdown completely when in mobile viewport. Right now the Bootstrap dropdown still opens in the background, and is then attempted to be hidden again via "d-none" class manually. 

Only thing that was in progress with my approach was the "toggling" in `_onViewportHasChanged`. I also deprecated the "hiddenClass" but without replacement with another "hide/show" class. Instead everything should happen via Bootstraps dropdown API directly without manual CSS classes.

Normally, the "show" class is set by Bootstrap itself "internally". But if this proposed solution here gets the job down, I'm also fine with it. 🙂 What do you think? @marcelbrode @divide29 

This was roughly my approach:

```diff
    _registerEventListeners() {
        this.el.addEventListener('click', this._onClickAccountMenuTrigger.bind(this, this.el));
+        this._dropdownWrapper.addEventListener('show.bs.dropdown', this._onClickPreventDropdown.bind(this));
```

```js

    /**
     * Prevent opening the Bootstrap dropdown in allowed viewports
     *
     * @param event
     * @private
     */
    _onClickPreventDropdown(event) {
        if (this._isInAllowedViewports() === true) {
            event.preventDefault();
        }
    }

```

Reply:
Yeah, i think your approach to prevent the event is the better one and i'm completly fine with it, thank you! :)

Reply:
Since @divide29 is fine with that, shall we close this?

Reply:
Thank you for your feedback. 👍 We will leave this PR open until the other solution is finished. If it doesn't work out we can still use your proposal @divide29.

Reply:
Hi, sorry for the long delay. The discussed solution has been merged now: https://github.com/shopware/shopware/commit/5a4705028ba36123856a5f28b344cfb989520bf2